app/frontend/terrier/toasts.ts

```diff
@@ -7,6 +7,7 @@ const log = new Logger('Toasts')
 export type ToastOptions = {
     color: ColorName
     icon?: IconName
+    duration?: 'long'
```

Comment:
I think maybe make this

```suggestion
    duration?: 'long' | 'default' | number
```

By allowing a specific number of milliseconds, we minimize the likelihood that we have to come back here to add a new option and update the version number for a one-off, but allowing specific named values makes consistency easier.